docs/getting_started/dashboard/work_with_dashboard.md

```diff
@@ -47,11 +47,11 @@ C. Buttons:
 |Redo|![Redo](dashboard_redo.png)|Redo latest change.|
 |Structure view|![Structure view](dashboard_structure_view.png)|Access Structure view toolbar.|
 |Dashboard blocks|![Dashboard blocks](dashboard_blocks.png)|Access Dashboard blocks toolbar.|
-|View switch|![View switch](dashboard_switch_toolbar.png)|Move Elements / Structure toolbar to the other side of the screen.|
+|View switch|![View switch](dashboard_switch_toolbar.png)|Move toolbar to the other side of the screen.|
```

Comment:
```suggestion
|View switch|![View switch](dashboard_switch_toolbar.png)|Move menu to the other side of the screen.|
```
I am wondering where the "View switch" name comes from. The name does not really reflect its function. (Not a review comment)

Reply:
Following Ibexa Design System - this element is called 'toolbox', so I'll update it.